catalog/conformance-tests/result-carboledger-001.json

```
@@ -0,0 +1,16 @@
+{
+    "tested_solution": {
+      "solution_id": "carboledger",
+      "version": "1.0.0"
```

I believe I see now what @shantanu1singh meant when he suggested replacing `"2.0.0"` by `"1.0.0"`: there's an inconsistency between the `solution.json` and the conformance test results. 
Since the `solution.json` actually identifies the solution, I would say that the conformance test results are the ones that should be updated, but correct me if I'm wrong.
```suggestion
      "version": "2.0.0"
```